### PR TITLE
CORS, ember connection

### PR DESCRIPTION
**Pull request by Easiv**
- Added CORS gem which allows rails to serve as backend api to ember

### CODE REVIEW

**Comment by Rxbsxn** [comment]
@Easiv 🔴

File: config/application.rb [hunk 1]
```diff
@@ -19,17 +19,15 @@
...
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
 
-    # Only loads a smaller set of middleware suitable for API only apps.
-    # Middleware like session, flash, cookies can be added back manually.
-    # Skip views, helpers and assets when generating a new resource.
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*', headers: :any, methods: [:get, :post, :options]
```
Comment (Rxbsxn) [inline config/application.rb]: what about updating and deleting resources?

**Comment by Easiv** [comment]
build is now 💚

**Comment by Easiv** [comment]
@Rxbsxn Should be good now

File: .circleci/config.yml [hunk 2]
```diff
@@ -6,16 +6,20 @@ version: 2
...
-       - image: circleci/ruby:2.4.1-node-browsers
-      
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
-
+      - image: circleci/ruby:2.5.1-node-browsers
```
Comment (PoslinskiNet) [inline .circleci/config.yml:9]: 👍

**Review by Rxbsxn — APPROVED** [review]

**Pull request merged**